testlibrary/src/main/java/com/example/testlibrary/LibraryActivity.java

```diff
@@ -55,8 +65,4 @@ protected void onDestroy() {
         super.onDestroy();
         System.out.println("LibraryActivity: " + "onDestroy() called");
     }
-
-    @Override
-    public void onBackPressed() {
-    }
```

Comment:
By removing this method, the Android OS will finish the Activity with an `Activity.RESULT_CANCELED` return code when the back button is pressed.